recipe/meta.yaml

```diff
@@ -12,9 +12,9 @@ source:
   sha256: 6f871ab07657becb3cca158dc4af96a1a14c0a360add04664fba325da43537fa
 
 build:
-  number: 203
+  number: 204
   # We lack openblas on Windows, and therefore can't build numpy there either currently.
-  skip: true  # [win or np!=111]
+  skip: true  # [win or np==110]
```

Comment:
Actually can this be changed to `np!=112` and then re-rendered. We have already dropped NumPy 1.10 with `conda-smithy` 2.0.0. Also we can't build two NumPys at once on CircleCI so we have to pick a version to build.